.storybook/styles/chromatic.css

```diff
@@ -5,5 +5,5 @@
 *,
```

Comment:
Looking at the preview I saw two issues.
1. The negative button still has the focus background color when clicking:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/14ef20f0-e837-4482-a4fc-3b11afa5191f)
2. The following seems to be a secondary button, but it's using the primary button checked state:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/7d7fcce4-71e6-4971-a4c7-74c67b7dac98)